templates/footer.inc.php

```diff
@@ -19,6 +19,11 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  *
  */
+
+$ampache_path = dirname(__FILE__);
+$prefix = realpath($ampache_path . "/../");
+require_once $prefix . '/lib/init.php';
```

Comment:
Why do you have these lines in footer.inc.php? Does this was for test and get committed by error?